v2/apiserver/internal/core/kubernetes/substrate_test.go

```diff
@@ -788,6 +788,65 @@ func TestSubstrateStartWorker(t *testing.T) {
 	}
 }
 
+func TestSubstrateStoreJobEnvironment(t *testing.T) {
```

Comment:
Wondering where we could/should put a test vetting the redaction of the environment for a job (and any sidecar containers)?  Although the logic hits the jobs store as well as the substrate, would it be possible to add unit test coverage using mocks?

Reply:
Good question. If the tests don't cover it already, it's an oversight. I'll fix that.

Reply:
It shouldn't be in this test suite. It should be in the tests for the JobsService. We can add assertions to the mock substrate to verify that redaction had taken place as expected on the Job that the JobsService passed to it.

Working on it.

Reply:
Fixed in c96370e090be8986e782f216eeda73c5b8709efc